openupgrade_scripts/scripts/website_event_questions/14.0.1.1/pre-migration.py

```diff
@@ -0,0 +1,27 @@
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    openupgrade.rename_tables(
+        env.cr,
+        [("event_answer", "event_question_answer")],
+    )
```

Comment:
also should call rename_models

Reply:
but don't delete the rename_tables. you have to put both! haha

Reply:
> but don't delete the rename_tables. you have to put both! haha

Haha I don't understand why should I do that?
You can explain for me🙏

Reply:
rename_models rename the models, and rename_tables rename the tables of those models. they are two different things and both are needed.

Reply:
I thought the system would understand by itself and only need 1 run.  It will update both